.github/workflows/integration_tests.yml

```diff
@@ -0,0 +1,32 @@
+name: Integration tests
+on:
+  pull_request: {}
+  push:
+    paths-ignore:
+    - '*.md'
+    - '**/*.md'
+    branches:
+    - master
+jobs:
+  kind_integration_tests:
```

Comment:
Let's chat about using KInD. I attempted to install KInD on an arm server and it required building k8s, as well.

I recall you saying that the Docker build portion would be fine on arm systems because the Dockerfile will detect the architecture. If `engineerd/setup-kind` will also detect the architecture, then we should be safe to use this